config/base/derive/src/proxy.rs

```diff
@@ -114,14 +132,15 @@ fn impl_load_from_disk(ast: &StructWithFields) -> proc_macro2::TokenStream {
                 serde_json::from_reader(reader)?
             }
         }
-    }
+    }.into()
 }
 
+// TODO: alter struct lvalues as well
```

Comment:
This might be really small, don't know if it's worth an issue. I just thought that as lvalue token streams are generated only once at structure creation time and stored as its fields, they might get desynchronized with the proxy struct when we alter it on the fly, like we do in this function.

That is not currently the case though, as no field idents are being changed, and neither types do in a form that would be relevant to this. So it seems this comment can be disregarded completely.